app/controllers/bulk_tagging/collections_controller.rb

```diff
@@ -0,0 +1,15 @@
+module BulkTagging
+  class CollectionsController < ApplicationController
```

Comment:
I will move this to a new ticket. I think it makes sense to reconsider the name when we add topics/browse pages too. We would be in a better place to rename this by then.